app/controllers/appeal-application/personal-details.ts

```diff
@@ -204,6 +211,78 @@ function postNationalityPage(updateAppealService: UpdateAppealService) {
   };
 }
 
+async function isOutOfCountryAppeal(req: Request) {
+  let appealOutOfCountry = req.session.appeal.appealOutOfCountry;
+  let outOfCountryFeatureEnabled: boolean = await LaunchDarklyService.getInstance().getVariation(req, FEATURE_FLAGS.OUT_OF_COUNTRY, false);
+  return (outOfCountryFeatureEnabled && appealOutOfCountry && appealOutOfCountry === 'Yes');
```

Comment:
@gavinslilley I think there is a duplicate `appealOutOfCountry` validation here 
(just tagging you here in Srini's absence as a second pair of eyes)